src/il_representations/envs/auto.py

```diff
@@ -195,6 +202,15 @@ def load_vec_env(benchmark_name, dm_control_full_env_names,
         assert final_env.observation_space.shape == (12, 64, 64), \
             final_env.observation_space.shape
         return final_env
+    elif benchmark_name == 'carla':
+        raw_carla_env = gym.make(gym_env_name)
+        raw_carla_env = CarlaImageObsEnv(raw_carla_env)
+        raw_carla_env = VecMonitor(venv=raw_carla_env, filename=None,
+                                            keep_buf=100)
+        final_env = VecFrameStack(raw_carla_env, 4, 'first')
```

Comment:
Is frame stacking used in the D4RL paper? I thought their default datasets only had 48*48*3 observations, not 48*4*12.

Reply:
Also, I noticed that `load_dataset_carla` takes a `carla_frame_stack` config var. If that's the case, then this function should use `carla_frame_stack` instead of a constant of 4.